core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1316,6 +1316,46 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_NETWORK_REQUEST_TIMEOUT_MS =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_NETWORK_REQUEST_TIMEOUT_MS)
+          .setDefaultValue("5sec")
```

Comment:
Should the default request timeout be longer? Looks like under 5sec, we are easy to get request timeout especially when reading backup/catching up.

Reply:
Currently It's consistent with the default leader election timeout. Bigger value will cause leadership changes to stall.